packages/apps/.eslintrc

```diff
@@ -4,5 +4,6 @@
     "oclif-typescript"
   ],
   "rules": {
+    "camelcase": "off"
```

Comment:
I'm not a huge fan of turning this rule off entirely, but I've experienced how annoying it is in a few of these packages, so I understand.